src/components/Home.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="homepage">
+    <h1>Welcome to Kiva 2049</h1>
```

Comment:
I'm a bit more optimistic that 2049 ;)
Also legal says we can't use Kiva yet - which is also why I'm doing it in my own repo rather than on of kiva's.